.github/workflows/email.yml

```diff
@@ -78,8 +78,10 @@ jobs:
               Author: ${{ env.AUTHOR}} 
               Link: ${{github.event.pull_request._links.html.href}}              
               Log Message: 
-                       ${{env.MERGE}} 
-              Compare: ${{env.COMPARE_URL}} 
+              ${{env.MERGE}} 
+
+              Compare: ${{env.COMPARE_URL}}
+
```

Comment:
The formatting here is the only real change.   Can we revert this to match the original?

Reply:
@tzinsky, the formatting is to 
1. Remove tabs before merge message 
2. Add a newline between merge message and compare.

I ran this formatting change in test repo and this is how the body looks. 
Log Message: 
  Merge pull request #38 from bhavanijayakumaran/format-1
  
  Update sendmail.yml 
  
  Compare:  https://github.com/bhavanijayakumaran/workflow/compare/ea7565f2a0[18](https://github.com/bhavanijayakumaran/workflow/actions/runs/4243611865/jobs/7376587121#step:7:18)fd5244a481029b1[22](https://github.com/bhavanijayakumaran/workflow/actions/runs/4243611865/jobs/7376587121#step:7:22)24fcdef44fd...ea7565f2a018fd5244a481029b12224fcdef44fd
  
  Diff:
            
           https://github.com/bhavanijayakumaran/workflow/pull/38.diff